packages/trader/src/Modules/Trading/Components/Elements/payout-per-point-mobile.tsx

```diff
@@ -24,8 +24,8 @@ type TProposalInfo = {
 };
 
 const PayoutPerPointMobile = observer(() => {
-    const { currency, proposal_info, contract_type, vanilla_trade_type } = useTraderStore();
-    const contract_key = isVanillaContract(contract_type) ? vanilla_trade_type : contract_type?.toUpperCase();
```

Comment:
After `vanilla_trade_type` was removed and replaced by `contract_type`, `Vanilla` can be treated the same way as other contract (e.g. `Turbos`)